app/models/session.rb

```diff
@@ -50,6 +53,9 @@ def setup_defaults
   def within_maximum_ttl
     time_remaining = (@created_at + MAX_SESSION_LIFETIME - Time.now.utc).round
     if time_remaining.negative?
+      log_message_to_sentry(
+        'Maximum Session Duration Reached', :info, {}, session_token: self.class.obscure_token(@token)
```

Comment:
why tag_context vs extra or user context for the token?

Reply:
`extra_context` gets added to the individual event 
![image](https://user-images.githubusercontent.com/3077884/36986899-084a93c4-2069-11e8-8b45-0e5be532a9ae.png)
and can only be seen when you drill into the specific event.  `tags_context` OTOH, get's _tallied_ and can be viewed on the [tags page for a given event type](http://sentry.vetsgov-internal/vets-gov/platform-api-production/issues/756/tags/)
![image](https://user-images.githubusercontent.com/3077884/36986974-417e6422-2069-11e8-8d0b-b88fef9f8197.png)

`user_context` confuses me, and I'm not clear exactly how `Sentry` uses this.